### PR TITLE
Run black.

### PR DESCRIPTION
**Pull request by AlbertMitjans**
These changes come from running:
```
black -l 100 pennylane/
```
and
```
black -l 100 tests/
```

### CODE REVIEW

File: tests/interfaces/test_jax.py [hunk 1]
```diff
@@ -338,7 +344,13 @@ def cost(a, cache):
...
-            return execute([tape], dev, gradient_fn=param_shift, cache=cache, interface=interface,)[
+            return execute(
+                [tape],
+                dev,
+                gradient_fn=param_shift,
+                cache=cache,
+                interface=interface,
+            )[
                 0
             ][0]
```
Comment (dwierichs) [inline tests/interfaces/test_jax.py:355]: What's going on here? Could removing the trailing comma in the old version change the behaviour of black to not do this? Or do we prefer this?

**Review by dwierichs — COMMENTED** [review]
All changes except for the one I commented on seem a nice update to me personally :)

**Review by AmintorDusko — APPROVED** [review]
Thank you for that @AlbertMitjans!

File: tests/interfaces/test_jax.py (hunk 1 above)
Comment (AlbertMitjans) [reply tests/interfaces/test_jax.py:355]: Removed the last comma!

Comment (AmintorDusko) [reply tests/interfaces/test_jax.py]: Much better now!

**Pull request merged**